### ISSUE TITLE
Scrolling is buggy in big documents

### ISSUE THREAD

**Issue by ujvaljoshi** [fileability/chocolat-public#1723]
OS X version: 10.10.2
Chocolat version: 3.1.4

When i am working in big html documents like 1000 lines. Scroll automatically reach end of file while scrolling.

**Comment by InerziaSoft** [comment]
Same here, but also with not so big documents (around 400 lines); it reaches the end if I use trackpad scrolling (scroll and then raise fingers from the trackpad). If I scroll very slowly, it doens't happen.

**Comment by ujvaljoshi** [comment]
Yeah its disappointing. Its been 4 months since any update on chocolat. I still face this issue.
